dynawo/sources/Modeler/Common/DYNConnector.cpp

```diff
@@ -147,16 +147,30 @@ ConnectorContainer::mergeFlowConnector() {
   // order flow connectors
   flowConnectors_.clear();
   flowConnectorByVarNum_.clear();
+  FlowAliasNameToFictitiousVarNum_.clear();
   list<shared_ptr<Connector> > flowConnectorsList;
   for (unsigned int i = 0; i < flowConnectorsDeclared_.size(); ++i) {
     shared_ptr<Connector> flowc(new Connector(*flowConnectorsDeclared_[i]));
     bool merged = false;
     for (vector<connectedSubModel>::iterator it = flowc->connectedSubModels().begin();
         it != flowc->connectedSubModels().end();
         ++it) {
-      const int numVar = it->subModel()->getVariableIndexGlobal(it->variable());
+      const boost::shared_ptr<Variable>& variable = it->variable();
+      int numVar;
+      if (variable->isAlias()) {
```

Comment:
I suppose there are more non alias variables than alias so I would switch the if and else statements ;)

Reply:
the cost is the same: one test of isAlias, so it will not change the performance IMHO